apriltag_ros/config/settings.yaml

```diff
@@ -1,11 +1,11 @@
 # AprilTag 3 code parameters
 # Find descriptions in apriltag/include/apriltag.h:struct apriltag_detector
 #                      apriltag/include/apriltag.h:struct apriltag_family
-tag_family:        'tag36h11' # options: tag36h11, tag25h9, tag16h5, tagStandard41h12
+tag_family:        'tag36h11' # options: tag36h11, tag25h9, tag16h5
```

Comment:
This change looks unrelated to other changes in this PR.  Please revert this change.